server/src/routes/api/v1/characterRouter.js

```diff
@@ -1,24 +1,35 @@
 import express from "express"
 
-import { Character } from "../../../models/index.js"
+import characterVoteRouter from "./characterVoteRouter.js"
+import { Character, Vote } from "../../../models/index.js"
 import cleanUserInput from "../../../services/cleanUserInput.js"
 
 const characterRouter = new express.Router()
 
-characterRouter.get("/", async (req,res) => {
+characterRouter.get("/", async (req, res) => {
   try {
     const characters = await Character.query()
     return res.status(200).json({ characters })
-  } catch(err) {
+  } catch (err) {
     return res.status(500).json({ errors: err })
   }
 })
 
-characterRouter.get("/:id", async (req,res) => {
-  try{
-    const character = await Character.query().findById(req.params.id)
+characterRouter.get("/:id", async (req, res) => {
+  const id = req.params.id
+  try {
+    const character = await Character.query().findById(id)
+    character.votes = await Vote.query().where("characterId", "=", id).sum("voteValue")
+    if (req.user) {
+      const userVote = await Vote.query().where("characterId", "=", id).where("userId", "=", req.user.id)
+      if (userVote[0]) {
+        character.voted = userVote[0].voteValue
+      }
+    } else {
+      character.voted = false
```

Comment:
why do we need both a boolean value AND the total `votes` attached to the character? Wouldn't the `votes` sum tell us if there are any votes?

Reply:
That was a weird result of a merge conflict.  We need the sum of the votes for the show page, and the boolean was originally just to check if the user had voted; it is now the value of the vote.